### DISCUSSION TITLE
Donations using IBAN?

### DISCUSSION THREAD

**Discussion by genodeftest** [FossifyOrg/General-Discussion#115] [General]
It would be nice if it would be possible to donate via the [IBAN](https://en.wikipedia.org/wiki/International_Bank_Account_Number) system to avoid fees or giving out personal data. I couldn't find an IBAN on the [donation page](https://github.com/FossifyOrg?q=&type=all&language=&sort=#support-fossify-heart).

(I know that this might not be possible for everyone. If it is not possible, please close this discussion.)

**Comment by inson1** [comment]
I dont think India has IBAN

https://juno.finance/blog/ifsc-swift-iban-difference
https://www.bankbazaar.com/currency-exchange/what-is-iban-number.html

swift has fees


Just use other options :) But Im not from India so idk (but naveensingh is)

**Reply by naveensingh** [reply]
> I dont think India has IBAN

That's right. SWIFT codes are used for wire transfers but there are fees. I'm not sure how much.

@genodeftest please consider other options :)
